types/src/on_chain_config/diem_version.rs

```diff
@@ -24,5 +24,10 @@ pub const DIEM_VERSION_2: DiemVersion = DiemVersion { major: 2 };
 //  - Multi-agent transactions
 pub const DIEM_VERSION_3: DiemVersion = DiemVersion { major: 3 };
 
+// NOTE: version number for release 1.4 of Diem
+// Items gated by this version number include:
+//  - Conflict-Resistant Sequence Numbers
+pub const DIEM_VERSION_4: DiemVersion = DiemVersion { major: 4 };
+
 // Maximum current known version
-pub const DIEM_MAX_KNOWN_VERSION: DiemVersion = DIEM_VERSION_3;
+pub const DIEM_MAX_KNOWN_VERSION: DiemVersion = DIEM_VERSION_4;
```

Comment:
@runtian-zhou Just want to make sure that this matches the on-chain DiemVersion numbers for upcoming releases.  

IIUC the upcoming release that's going through testing will have DiemVersion number 3, and the next release (the one this will be in) will have DiemVersion 4.

Reply:
That's correct!